src/pages/BrowseSchemes.svelte

```diff
@@ -135,78 +48,19 @@
     }
   }
 
-  function addSchemeToSidebar(gj: GeoJSON) {
-    let byScheme = {};
-
-    // Assume the input has a top-level dictionary keyed by scheme_reference
-    for (let [scheme_reference, scheme] of Object.entries(gj.schemes)) {
-      byScheme[scheme_reference] = {
-        scheme_reference,
-        num_features: 0,
-        ...scheme,
-      };
-    }
-
-    for (let feature of gj.features) {
-      byScheme[feature.properties.scheme_reference].num_features++;
-    }
-
-    schemes = Object.values(byScheme);
-
-    let set1 = new Set();
-    let set2 = new Set();
-    for (let x of schemes) {
-      if (x.authority_or_region) {
-        set1.add(x.authority_or_region);
-      }
-      if (x.funding_programme) {
-        set2.add(x.funding_programme);
-      }
-    }
-    authorities = Array.from(set1.entries());
-    authorities.sort();
-    fundingProgrammes = Array.from(set2.entries());
-    fundingProgrammes.sort();
-  }
-
   function tooltip(props: { [name: string]: any }): string {
     // TODO Move into a Svelte component, so we don't have to awkwardly build up HTML like this
-    var html = `<table>`;
-    for (let [key, value] of Object.entries(props)) {
-      html += `<tr><td>${key}</td><td>${value}</td></tr>`;
+    var html = `<div class="govuk-prose" style="max-width: 30vw;">`;
```

Comment:
This feels kinda weird. Looks like we're doing what Svelte is supposed to do natively in ts?

Reply:
You're right. At the moment, it's a bit convoluted to make maplibre popups work with Svelte, because it needs this `setHTML` call. A nicer approach using `svelte-maplibre` looks [like this](https://github.com/dabreegster/geodiffr/blob/76c4f596a0659c1c9a192314063e117b842371ac/src/App.svelte#L142), where the popup component is nested logically under the layer, which is also expressed as a Svelte component. We're not using that library yet here (and it'll take some work to migrate to it, and I'm not sure it's the right move yet).

Alternatively, I've done a simpler hack in https://github.com/a-b-street/osm2streets/blob/147de2ec76aeb8397cb813dab8572a151a1eb7e8/web/src/street-explorer/Popups.svelte#L26 of instantiating a Svelte component manually. Should we do the same here?

Reply:
Alright, Either of those does seem nicer than this, but this PR isn't changing our approach so I don't think this should hold up this PR, approving